ToDoTakeTWo/app/src/main/java/com/korbkenny/todotaketwo/MainActivity.java

```diff
@@ -0,0 +1,197 @@
+                LayoutInflater theInflater = LayoutInflater.from(context);
+                View thePrompt = theInflater.inflate(R.layout.dialog_remove, null);
+                AlertDialog.Builder alertDialogBuilder = new AlertDialog.Builder(context);
+
+                alertDialogBuilder.setView(thePrompt);
+
+                alertDialogBuilder
+                        .setPositiveButton("Sure",
+                                new DialogInterface.OnClickListener() {
+                                    public void onClick(DialogInterface dialog, int id) {
+                                        bigOldListOfLists.getListOfLists().remove(position);
+                                        mArrayAdapter.notifyDataSetChanged();
+                                    }
+                                })
+                        .setNegativeButton("Nah",
+                                new DialogInterface.OnClickListener() {
+                                    public void onClick(DialogInterface dialog, int id) {
+                                        dialog.cancel();
+                                    }
+                                });
+
+                AlertDialog alertDialog = alertDialogBuilder.create();
+                alertDialog.show();
+                return true;
+            }
+        });
+
+
+        //ARRAY ADAPTER
+        mArrayAdapter = new ArrayAdapter<ToDoList>(context, android.R.layout.simple_list_item_1, bigOldListOfLists.getListOfLists());
+        mListView.setAdapter(mArrayAdapter);
+
+    }
+}
+//}
+
+
+/**
+ *
+ *
+ *      ~~~~~~~~~~~~~~  Below is random snippets of code I scrapped (But may want in the future? [prob not])
+ *
+ */
+
+
+
+//if (!(bigOldListOfLists.getListOfLists().equals(null))) {
+// mListView.setAdapter(new BaseAdapterMain(context, bigOldListOfLists.getListOfLists()));
+// }
+
+//        bigOldListOfLists.getListOfLists().add(new ToDoList("Hello there"));
+//        bigOldListOfLists.getListOfLists().add(new ToDoList("HIIIIii"));
+
+
+//        mBaseAdapter = new BaseAdapter() {
+//            @Override
+//            public int getCount() {
+//                return (bigOldListOfLists.getListOfLists().size() == 0) ? 0 : bigOldListOfLists.getListOfLists().size();
+//            }
+//
+//            @Override
+//            public Object getItem(int position) {
+//                return bigOldListOfLists.getListOfLists().get(position);
+//            }
+//
+//            @Override
+//            public long getItemId(int position) {
+//                return 0;
+//            }
+//
+//            @Override
+//            public View getView(int position, View convertView, ViewGroup parent) {
+//                if(convertView==null){
+//                    convertView = getLayoutInflater().inflate(R.layout.list_lists_item_main, parent, false);
+//                }
+//                ((TextView) convertView).setText(getItem(position));
```

Comment:
I know you didn't end up using this code, but I think the problem was that `convertView` represents the whole inflated view, so in this case the LinearLayout with a child TextView, not just the TextView. So when you cast it to TextView it's going to cause a problem. Instead, try `convertView.findViewById(R.id.OneList).setText(...);`